packages/css/src/fonts.css

```diff
@@ -35,9 +28,11 @@
 /**
  * Legacy font-face with an own family for each font
  */
+
+/* Posten Sans Light is deprecated, map the references to Posten Sans Medium */
 @font-face {
   font-family: "Posten Sans Light";
```

Comment:
Is there a reason to keep the Posten Sans Light font definition?

Reply:
I mean, we are before 1.0

Reply:
Intention here was if HDS is used in combination with a hedwig legacy version not updated, this would catch the usage of PS Light and patch it.

But 🤷 Can remove it